json_object.c

```diff
@@ -111,15 +195,23 @@ static void json_object_fini(void)
 
 /* helper for accessing the optimized string data component in json_object
  */
-static const char *get_string_component(const struct json_object *jso)
+static inline char *get_string_component_mutable(struct json_object *jso)
+{
+	if (JC_STRING_C(jso)->len < 0)
+	{
+		/* Due to json_object_str_string(), we might have a pointer */
```

Comment:
modify:   str--->set   /* Due to json_object_set_string(), we might have a pointer */

Reply:
👍   Will fix.

Reply:
See commit 4c107121